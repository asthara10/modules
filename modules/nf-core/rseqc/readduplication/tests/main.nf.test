nextflow_process {

    name "Test Process RSEQC_READDUPLICATION"
    script "../main.nf"
    process "RSEQC_READDUPLICATION"

    tag "modules"
    tag "modules_nfcore"
    tag "rseqc"
    tag "rseqc/readduplication"

    test("test_rseqc_readduplication") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end: false ], // meta map
              file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
            ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
