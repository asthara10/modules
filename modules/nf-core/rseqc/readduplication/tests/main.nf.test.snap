{
    "versions": {
        "content": [
            [
                "versions.yml:md5,06a70ece474de57fc1dc7585e7cfd5a0"
            ]
        ],
        "timestamp": "2023-11-24T12:40:43.116511"
    }
}